src/GraphMetadataViewExtension/GraphMetadataView.xaml

```diff
@@ -72,68 +124,100 @@
+                                </Binding.Converter>
+
+                            </Binding>
+                        </TextBox.Text>
+                    </TextBox>
+                </GroupBox>
+            </StackPanel>
+                
+            <Separator Background="{StaticResource FilterIconColor}" Margin="0,20,0,20"/>
         </StackPanel>
 
         <GroupBox Grid.Row="1"
-                  Header="Custom Properties">
+                  Header="Custom Properties" Style="{StaticResource CustomPropBox}">
             <GroupBox.Template>
                 <ControlTemplate TargetType="{x:Type GroupBox}">
                     <Grid SnapsToDevicePixels="true">
                         <Grid.RowDefinitions>
                             <RowDefinition Height="Auto" />
                             <RowDefinition Height="Auto" />
                         </Grid.RowDefinitions>
-                        <Grid Margin="9,1,9,0"
+                        <Grid Margin="10,0"
                               Grid.Row="0">
                             <Grid.ColumnDefinitions>
                                 <ColumnDefinition Width="*" />
                                 <ColumnDefinition Width="20" />
                             </Grid.ColumnDefinitions>
-                            <ContentPresenter ContentSource="Header" />
-                            <Button Grid.Column="1">
+                            <ContentPresenter ContentSource="Header"/>
+                            <Button Grid.Column="1" Command="{Binding AddCustomPropertyCommand}">
                                 <Button.Template>
                                     <ControlTemplate>
                                         <fa:ImageAwesome Icon="Plus"
-                                                         VerticalAlignment="Stretch" />
+                                                         VerticalAlignment="Stretch" 
+                                                         Foreground="{StaticResource FilterIconColor}"/>
                                     </ControlTemplate>
                                 </Button.Template>
                             </Button>
 
                         </Grid>
-                        <ContentPresenter Margin="9,1,9,0"
+                        <ContentPresenter Margin="0,0,0,0"
                                           Grid.Row="1"
-                                          SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" />
+                                          SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}"/>
                     </Grid>
 
                 </ControlTemplate>
 
             </GroupBox.Template>
-
-            <StackPanel Orientation="Vertical">
-                <ListView HorizontalContentAlignment="Stretch"
+            <StackPanel Orientation="Vertical" ScrollViewer.VerticalScrollBarVisibility="Visible">
+                <ScrollViewer VerticalScrollBarVisibility="Auto" Height="440">
```

Comment:
We should not hardcode the height for the viewer. As this is in a Grid Row i believe it should automatically fill up the entire space, does that not happen?

Reply:
Done.